From jrm8005@gmail.com  Fri Oct  3 11:11:03 2008
From: "J.R. Mauro" <jrm8005@gmail.com>
Date: Fri, 3 Oct 2008 12:21:49 -0400
Subject: Staging: Fix gcc warnings in sxg
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Message-ID: <3aaafc130810030921l79ee87a9k6cef0132b2c5b715@mail.gmail.com>
Content-Disposition: inline


Fix for compiler warning about format specifier in prints in
drivers/staging/sxg/sxg.c
Prints were using %x to print unsigned long data.


Signed-off-by: J.R. Mauro <jrm8005@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sxg/sxg.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/staging/sxg/sxg.c
+++ b/drivers/staging/sxg/sxg.c
@@ -453,7 +453,7 @@ static int sxg_allocate_resources(p_adap
 	// fails.  If we hit a minimum, fail.
 
 	for (;;) {
-		DBG_ERROR("%s Allocate XmtRings size[%x]\n", __FUNCTION__,
+		DBG_ERROR("%s Allocate XmtRings size[%lx]\n", __FUNCTION__,
 			  (sizeof(SXG_XMT_RING) * 1));
 
 		// Start with big items first - receive and transmit rings.  At the moment
@@ -470,7 +470,7 @@ static int sxg_allocate_resources(p_adap
 		}
 		memset(adapter->XmtRings, 0, sizeof(SXG_XMT_RING) * 1);
 
-		DBG_ERROR("%s Allocate RcvRings size[%x]\n", __FUNCTION__,
+		DBG_ERROR("%s Allocate RcvRings size[%lx]\n", __FUNCTION__,
 			  (sizeof(SXG_RCV_RING) * 1));
 		adapter->RcvRings =
 		    pci_alloc_consistent(adapter->pcidev,
@@ -531,7 +531,7 @@ static int sxg_allocate_resources(p_adap
 		return (STATUS_RESOURCES);
 	}
 
-	DBG_ERROR("%s Allocate EventRings size[%x]\n", __FUNCTION__,
+	DBG_ERROR("%s Allocate EventRings size[%lx]\n", __FUNCTION__,
 		  (sizeof(SXG_EVENT_RING) * RssIds));
 
 	// Allocate event queues.
@@ -562,7 +562,7 @@ static int sxg_allocate_resources(p_adap
 	}
 	memset(adapter->Isr, 0, sizeof(u32) * IsrCount);
 
-	DBG_ERROR("%s Allocate shared XMT ring zero index location size[%x]\n",
+	DBG_ERROR("%s Allocate shared XMT ring zero index location size[%lx]\n",
 		  __FUNCTION__, sizeof(u32));
 
 	// Allocate shared XMT ring zero index location
